WcaOnRails/app/views/static_pages/disclaimer.html.erb

```diff
@@ -3,11 +3,5 @@
 <div class="container">
```

Comment:
The `react_component` call automatically creates a `div` that houses the to-be-rendered component. Its HTML properties can be controlled by passing a second additional parameter to the method like so:
```ruby
react_component('MyComponent', {
  'someProp': true,
  'otherProp': 123
}, {
  id: 'html-id',
  class: 'container'
})
```

Reply:
Ah so no need for this div then